website/static/_redirects

```diff
@@ -1,6 +1,7 @@
-# Redirects from what the browser requests to what we serve.
-#
+/docs/latest/* /docs/v0.10/:splat 302
```

Comment:
👍🏻 nice. Works?

Reply:
They only thing about this now is that if we ever change the layout of the docs, all the `latest` URLs will need to be updated. Not a big deal, but we should probably make not of that here?

Reply:
Those a temporary redirects, not permanent ones as they were before. I think we can use some existing service to monitor 404, or just write our own crawler that would detect broken link